src/ratelimitmongo.d.ts

```diff
@@ -0,0 +1,25 @@
+
+declare module 'rate-limit-mongo'
```

Comment:
why this ?

Reply:
The typings of rate-limit-mongo were not available so we need to create the typings file ourselves